src/redux/actions.ts

```diff
@@ -7,6 +7,48 @@
 
 import { PARAM_LANGUAGE } from '../utils/config-params';
 import { Action } from 'redux';
+import { UserManagerState } from '../routes';
+
+export const UPDATE_USER_MANAGER_STATE = 'UPDATE_USER_MANAGER_STATE';
+export type UserManagerAction = Readonly<
+    Action<typeof UPDATE_USER_MANAGER_STATE>
+> & {
+    userManager: UserManagerState;
+};
+export function updateUserManager(
+    userManager: UserManagerState
+): UserManagerAction {
+    return { type: UPDATE_USER_MANAGER_STATE, userManager };
+}
+export function updateUserManager_(
```

Comment:
I never saw a function ending with an underscore, why and when we should do that ?

Reply:
because I wanted to do twice the function with different arguments, like with Java, but JS & TS don't permit that...